profiles/lib/destination.lua

```diff
@@ -0,0 +1,27 @@
+local Destination = {}
+
+function Destination.get_destination(way)
+  local destination = way:get_value_by_key("destination")
+  local destination_ref = way:get_value_by_key("destination:ref")
```

Comment:
In theory we could make use of some less-popular tags here, too:
http://taginfo.openstreetmap.org/search?q=destination

And some of the proposed features:
http://wiki.openstreetmap.org/wiki/Proposed_features/Destination_details

But for a start this should give of the most popular ones.